### DISCUSSION TITLE
Jump to source failed

### DISCUSSION THREAD

**Discussion by lakako** [lexical-lsp/lexical#613] [General]
project url: https://github.com/roylez/cloak

After I change mix.exs:7 to 'version: "1.11"'. Jump to source work properly.

Is this a bug?

**Comment by scohen** [comment]
No, this is not a bug. Lexical supports elixir 1.13 and newer. 1.11 is not supported.
https://github.com/lexical-lsp/lexical/blob/main/pages/installation.md#caveats

**Reply by lakako** [reply]
sorry for my incorrect description.

1.11 means project version. not elixir version

Editor is vscode 1.86.2

on that project lexical jump to source not working

https://github.com/roylez/cloak/blob/5a7671a401512fbf4dab946a88c2b7acd6081f9e/mix.exs#L7

change this line to 

```
version:  "x.xx.x",
```

restart editor

Jump to source works

**Reply by scohen** [reply]
You sure? Lexical doesn't care about a project's version, but it definitely won't work on that version of elixir.
